srcpkgs/FeatherPad/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'FeatherPad'
 pkgname=FeatherPad
-version=1.2.0
+version=1.4.1
 revision=1
-build_style=qmake
-hostmakedepends="qt5-qmake qt5-host-tools pkg-config"
```

Comment:
still needs qmake here

Reply:
Forgot about the PR... Thanks, fixed, rebased and pushed.